Documentation/DocuBlocks/Rest/Transactions/post_api_transaction_begin.md

```diff
@@ -23,8 +23,7 @@ Allow reading from undeclared collections.
 @RESTBODYPARAM{lockTimeout,integer,optional,int64}
 an optional numeric value that can be used to set a
 timeout for waiting on collection locks. If not specified, a default
-value will be used. Setting *lockTimeout* to *0* will make ArangoDB
-not time out waiting for a lock.
+value will be used. The maximum value for stream transactions is 30 seconds.
```

Comment:
So i do understand this as:
* lockTimeout = 0 means we do not wait? (please clarify 0 here)
* lockTimeout = 5 => we wait 5 sec (ok)
* lockTimeout = 55 => we wait 30 sec?

What is the defaultValue we use?

Reply:
In order to maintain something resembling backwards compatibility, we probably need to interpret an input of 0 as the max. I believe that's exactly what we do now, but we should probably document it here.

Reply:
60 seconds